### PR TITLE
Steve/custom domains

### PR DESCRIPTION
**Pull request by Steve-B-Murphy**
creating branch for XDN-14491

### CODE REVIEW

File: guides/production.md [hunk 1]
```diff
@@ -21,7 +21,11 @@ To configure your custom domains:
 
 If you're migrating to {{ PRODUCT_NAME }} from [Fastly](https://www.fastly.com/), you will need to do the following before adding your domains to your {{ PRODUCT_NAME }} environment:
 
+<<<<<<< HEAD
+- [Contact Fastly support](https://support.fastly.com/hc/en-us/requests/new?ticket_form_id=360000269711) and request that control of your domains be transferred to {{ PRODUCT_NAME }}. Be sure to explicitly list each domain that needs to transferred and ask Fastly to contact support(at){{ DOMAIN }} if they need {{ PRODUCT_NAME }} to confirm the transfer.
```
Comment (adebiyi-al) [inline guides/production.md:25]: Typo: "...needs to transferred" should be "...needs to be transferred"

File: guides/production.md [hunk 2]
```diff
@@ -174,7 +186,7 @@ __Note:__ If you already have an existing certificate, you can use it by skippin
 
 3. Add an `_acme-challenge.` CNAME DNS entry to allow {{ PRODUCT_NAME }} to issue a certificate request on your behalf.
 
-   Log into your DNS provider and add one `CNAME` type DNS entry with the value `_acme-challenge.<your-domain-here>` for each domain you use on your Layer0 website. For example, if your domain is `mywebsite.xyz`, the DNS entry should have a value of `_acme-challenge.mywebsite.xyz`. This record should point to `_acme-challenge.xdn-validation.com`. Repeat the operation of each domain associated with your Layer0 website.
+   Log into your DNS provider and add one `CNAME` type DNS entry with the value `_acme-challenge.<your-domain-here>` for each domains you use on your Layer0 website. For example, if your domain is `mywebsite.xyz`, the DNS entry should have a value of `_acme-challenge.mywebsite.xyz`. This record should point to `_acme-challenge.xdn-validation.com`. Repeat the operation of each domain associated with your Layer0 website.
```
Comment (adebiyi-al) [inline guides/production.md:189]: Typo: "...for each domains" should be "for each domain"

File: guides/production.md [hunk 3]
```diff
@@ -236,7 +248,7 @@ __Note:__ If you already have an existing certificate, you can use it by skippin
 
 TLS certificates are issued by Certificate Authorities (CA) based on Certificate Signing Request (CSR) that they receive from you. Alongside the CSR the same process creates the certificate's private key. You only need to share your CSR with CA, not the private key which you should store securely.
 
-The following steps describe the creation of the CSR and private key with OpenSSL. OpenSSL is an open-source toolkit for the TLS protocol. We recommend using OpenSSL because it ensures that your private key will only be stored locally on your infrastructure. Your CA may have more customized guides or an entirely customized certification process.
+The following steps describe the creation of the CSR and private key with OpenSSL. OpenSSL is an open-source toolkit for the TLS protocol. We recommend using OpenSSL because it ensures that your private key will only be stored locally on your infrastructure. Your CA may to have more customized guides or entirely customized certification process.
```
Comment (adebiyi-al) [inline guides/production.md:251]: Typo: "Your CA may to..."

File: guides/production.md (hunk 2 above)
Comment (adebiyi-al) [inline guides/production.md:189]: Typo: "...for each domains" should be "for each domain"

**Review by adebiyi-al — CHANGES_REQUESTED** [review]

**Comment by Steve-B-Murphy** [comment]
I'm making changes locally and will do another push.

**Pull request closed without merging**